lib/opal/cli.rb

```diff
@@ -131,6 +131,7 @@ def compiler_option_names
         inline_operators
         enable_source_location
         parse_comments
+        es_six_imexable
```

Comment:
can you also change `es-six-imexable` / `es_six_imexable` to `es6-modules` / `es6_modules` everywhere?